apps/shared/deno.sh

```diff
@@ -0,0 +1,23 @@
+DENO_MIN_VERSION="1.9.1"
+
+function denoInstall() {
+
+    { # try
+        echo "Deno version check:" && denoCmd upgrade --version $DENO_MIN_VERSION
+    } ||
+    { # catch
+        echo "Installing Deno..."
+        # just one line of command that works on all OSes
+        # (temporary cd into PATH_DEPS)
+        curl -fsSL https://deno.land/x/install/install.sh | DENO_INSTALL="$PATH_DEPS/deno" sh
```

Comment:
add -k

Reply:
Added as temporary Hotfix in 686535d7a4ddea809331ba7635d7193015fd2121